cli/command_policy_show.go

```diff
@@ -420,6 +420,17 @@ func appendActionsPolicyRows(rows []policyTableRow, p *policy.Policy, def *polic
 		rows = append(rows, policyTableRow{"No actions defined.", "", ""})
 	}
 
+	if isWindows() {
```

Comment:
we generally don't generate OS-specific output, one can be looking at a policy belonging to a different user/host, and output should not change whether we run on Windows or Linux.